src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3092,7 +3092,7 @@ public static void DCS_MyPersonSurrogate()
     [Fact]
     public static void DCS_FileStreamSurrogate()
     {
-        const string TestFileName = "Test.txt";
+        string TestFileName = Path.GetTempFileName();
```

Comment:
Adjusted the name of the local var. I'm not sure whether implementing an entire base class for this single instance test is worth the hassle - would prefer option A myself.